components/ecosystem/TestTable.vue

```diff
@@ -47,21 +46,32 @@
           fill="#c6c6c6"
           aria-label="No test information"
         />
-      </cv-data-table-cell>
-    </cv-data-table-row>
-  </AppDataTable>
+      </bx-table-cell>
+    </bx-table-row>
+  </UiAppDataTable>
 </template>
 
 <script setup lang="ts">
-import { TableRowElement } from "~/components/ui/AppDataTable.vue";
+import "@carbon/web-components/es/components/tooltip/tooltip-icon.js";
+import Information16 from "@carbon/icons-vue/lib/information/16";
+import CheckmarkFilled16 from "@carbon/icons-vue/lib/checkmark--filled/16";
+import ErrorFilled16 from "@carbon/icons-vue/lib/error--filled/16";
+import PendingFilled16 from "@carbon/icons-vue/lib/pending--filled/16";
+
+interface filteredDataItem {
```

Comment:
You can move this interface to the `types/ecosystem.ts` file, so we can have all the types together on the same place

Reply:
I'd personally prefer to keep type definitions in the only file that uses them. I see the value in putting them in a central place if they are used by multiple files and need to be imported. Otherwise I think it might lead to harder maintainability.